qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/controllers/CreateAffiliationController.groovy

```diff
@@ -0,0 +1,41 @@
+package life.qbic.portal.qoffer2.web.controllers
+
+import groovy.util.logging.Log4j2
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.datamodel.dtos.business.AffiliationCategory
+import life.qbic.portal.portlet.customers.affiliation.create.CreateAffiliationInput
+
+/**
+ * Controller class adapter from view information into use case input interface
+ *
+ * This class translates the information that was received from the view into method calls to the use case
+ *
+ * @since: 1.0.0
```

Comment:
I think we agreed on having the author tag as well?

Reply:
I think we agreed to leave it out